### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by ViktorLitviakov8050**

### CODE REVIEW

File: notes/randomNote/models.py [hunk 1]
```diff
@@ -34,6 +35,12 @@ class Note(models.Model, JSONSourceable):
     is_archived = models.BooleanField(default=False)
     labels = models.ManyToManyField(Label)
 
+    @classmethod
+    def random_note(cls):
+        id_array = cls.objects.values_list('id', flat=True)
+        random_id = choice(id_array)
```
Comment (zasviat) [inline notes/randomNote/models.py:47]: let's add here a check here to be sure that there's no error if array is empty
image

File: notes/seed.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
     raise Exception('Zip file path is not provided')
 zip_file_path = sys.argv[1]
 
-folder_to_extract_files_from = 'Takeout/Keep'
```
Comment (zasviat) [inline notes/seed.py:19]: let's change this back to 'Takeout/Keep' and move it to settings
`NOTES_SOURCE_DIR = os.path.join('Takeout', 'Keep')`

then we'll be able to reference this config as `settings.NOTES_SOURCE_DIR` in seed.py and in models.py to copy file
docs: https://stackoverflow.com/questions/7867797/how-do-i-reference-a-django-settings-variable-in-my-models-py

File: notes/randomNote/models.py [hunk 3]
```diff
@@ -56,6 +63,8 @@ def from_dict(cls, dictionary):
                 attachment = Attachment.from_dict(attachment_raw)
                 attachment.note = note
                 attachment.save()
+                copy_file(f"Keep/{attachment.file_path}",f"Attachments/{attachment.file_path}")
```
Comment (zasviat) [inline notes/randomNote/models.py]: we can use here config variables from settings.py
check another comment please

File: notes/seed.py [hunk 4]
```diff
@@ -16,7 +16,7 @@
...
+folder_to_extract_files_from = 'Keep'
 folder_with_extracted_files = os.path.join(os.getcwd())
```
Comment (zasviat) [inline notes/seed.py:22]: let's add code to delete temporary folder extracted from zip archive at the end of this file

```suggestion
from pathlib import Path
extracted_temporary_folder_path = Path(settings.NOTES_SOURCE_DIR).resolve().parent
shutil.rmtree(extracted_temporary_folder_path)
```

**Review by zasviat — CHANGES_REQUESTED** [review]
we need some changes due to comments

File: notes/randomNote/models.py (hunk 1 above)
Comment (ViktorLitviakov8050) [reply notes/randomNote/models.py:47]: Let's resolve it on demand)))

File: notes/seed.py (hunk 2 above)
Comment (ViktorLitviakov8050) [reply notes/seed.py:19]: resolved

**Review by zasviat — APPROVED** [review]
nice work)

**Pull request merged**